Doc/library/mailcap.rst

```diff
@@ -26,6 +26,16 @@ The mailcap format is documented in :rfc:`1524`, "A User Agent Configuration
 Mechanism For Multimedia Mail Format Information", but is not an internet
 standard.  However, mailcap files are supported on most Unix systems.
 
+.. _ailcap-shell-injection:
+
+.. warning::
+
+   The Python implementation is vulnerable to shell command injection. By
+   design, the mailcap format uses shell commands. The caller is responsible to
+   validate the filename. The caller can create temporary filename, using
```

Comment:
It's not only about the filename. The MIME type and parameter list need to be validated as well.
Also, how should they be validated? That's not clear.

Reply:
You're right, I rephrased the warning.

Reply:
"The caller is responsible to validate and sanitize input arguments."